### PR TITLE
Add Serilog logger file

### PR DESCRIPTION
**Pull request by nreal**
# Background
Add An Serilog logger [link](https://github.com/serilog/serilog/wiki/Getting-Started)

Could you review it?

### CODE REVIEW

File: Doppler.Jobs.Server/Startup.cs [hunk 1]
```diff
@@ -97,6 +99,12 @@ private static IAsyncPolicy<HttpResponseMessage> GetRetryPolicy(int retry)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
+            Log.Logger = new LoggerConfiguration()
+                .MinimumLevel.Information()
+                .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
+                .WriteTo.File("Logs\\DopplerJobs.log", rollingInterval: RollingInterval.Day)
+                .CreateLogger();
```
Comment (andresmoschini) [inline Doppler.Jobs.Server/Startup.cs:106]: Is it possible to also continue writing to _stdout_?

File: Doppler.Jobs.Server/appsettings.json [hunk 2]
```diff
@@ -1,12 +1,4 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information",
-      "Hangfire": "Warning"
-    }
-  },
```
Comment (andresmoschini) [inline Doppler.Jobs.Server/appsettings.json:9]: I like this declarative configuration in appsettings in place of code. Is it really necessary to use the code one?

File: Doppler.Jobs.Server/Startup.cs (hunk 1 above)
Comment (nreal) [reply Doppler.Jobs.Server/Startup.cs]: Yes done in this commit https://github.com/MakingSense/doppler-jobs/pull/3/commits/0734fe3ceedb6a0392995b4357de89dbb0d3850f

screen

File: Doppler.Jobs.Server/appsettings.json (hunk 2 above)
Comment (nreal) [reply Doppler.Jobs.Server/appsettings.json:9]: no, we can use this configuration for each transformation, I'll revert this change

Comment (nreal) [reply Doppler.Jobs.Server/appsettings.json]: Done in this commit https://github.com/MakingSense/doppler-jobs/pull/3/commits/884e90723c61163e11c5fd7433c4f2431195d09f

**Comment by andresmoschini** [comment]
❤️ 🚀

**Pull request merged**